src/note-components/Authentication/login.jsx

```diff
@@ -0,0 +1,91 @@
+import "../../public-css/root.css";
+import "../Authentication/auth.css";
+import { useAuth } from "../../Context/auth-context";
+import { loginAPI } from "../../note-API/auth-API";
+import { Navbar } from "../Navbar/navbar";
+import { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+
+const Login = () => {
+const { authDispatch} = useAuth();
+const navigate = useNavigate();
+const [ user, setUser ] = useState({email: "", password: ""});
+
+const guestUser = {
+email: "adarshbalika@gmail.com",
+password: "adarshBalika123",
+};
+
+const changeHandler = (e) => {
+const { id, value } = e.target;
+setUser({ ...user, [id]: value });
+};
+
+const guestUserHandler = (e) => {
+e.preventDefault();
+setUser(guestUser);
+};
+
+const loginHandler = async (e) => {
+e.preventDefault();
+if ((user.email !== "", user.password !== "")) {
```

Comment:
Suggestion: You can add required attribute in the email and password field instead of checking here.